src/mca/bfrops/base/bfrop_base_pack.c

```diff
@@ -336,7 +336,8 @@ pmix_status_t pmix_bfrops_base_pack_string(pmix_pointer_array_t *regtypes,
     for (i = 0; i < num_vals; ++i) {
         if (NULL == ssrc[i]) {  /* got zero-length string/NULL pointer - store NULL */
             len = 0;
-            if (PMIX_SUCCESS != (ret = pmix_bfrops_base_pack_int32(regtypes, buffer, &len, 1, PMIX_INT32))) {
+            PMIX_BFROPS_PACK_TYPE(ret, buffer, &len, 1, PMIX_INT32, regtypes);
```

Comment:
Move to the upper commit :
[ ](/karasevb)
` [bfrops: redefinition regtypes for internal usage](/pmix/pmix/pull/1130/commits/daef318771b1f537c49a0a63badeac5f1eae86f1)`